.github/CODEOWNERS

```diff
@@ -190,6 +190,7 @@ receiver/filereceiver/                                   @open-telemetry/collect
 receiver/filestatsreceiver/                              @open-telemetry/collector-contrib-approvers @atoulme
 receiver/flinkmetricsreceiver/                           @open-telemetry/collector-contrib-approvers @jonathanwamsley @djaglowski
 receiver/fluentforwardreceiver/                          @open-telemetry/collector-contrib-approvers @dmitryax
+receiver/gitproviderreceiver/                            @open-telemetry/collector-contrib-approvers @adrielp @astencel-sumo
```

Comment:
@atoulme, are we enforcing that new code owners have to be members of OTel?

Reply:
As soon as this pr is through, I'll be opening up a membership request. I'm currently looking for one more sponsor 😉 <cough><cough>